### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by joshhaynes** [playframework/play-grpc#463]
This hasn't been released in > 3 years -- any chance we can get an updated release along with updated dependencies?

**Comment by sentenza** [comment]
Same here. Also what's the plan for using gRPC in Scala Play 3 (with Pekko)?

**Comment by ihostage** [comment]
Hi, folks! 👋 
I'm sure that Play gRPC stays one of the important  modules in the Play platform. But, of course, we need a time for actualization and migrate it to Pekko. Or your help instead of our time 😉 
For example, right now migrate to Pekko is blocked by #471, would you be keen to provide a pull request for that?

**Comment by sentenza** [comment]
> Or your help instead of our time 😉 For example, right now migrate to Pekko is blocked by #471, would you be keen to provide a pull request for that?

I've just opened a new PR for removing Lagom. I hope this helps

#477

**Comment by ihostage** [comment]
Thank a lot, Alfredo! 👍 I will check it as soon as possible after finishing my activities are related with Gradle support.

**Comment by sentenza** [comment]
@ihostage @mkurz as soon as the new versions for Play 2.9 (v0.11) and Play 3 (v1) are out this issue can be closed

**Comment by ihostage** [comment]
> @ihostage @mkurz as soon as the new versions for Play 2.9 (v0.11) and Play 3 (v1) are out this issue can be closed

Yep 👌 But version for Play 3 will be `0.12.0`, not `1.0.0` 😉 We aren't ready yet to say that this module has a final API 😄

**Comment by mkurz** [comment]
@ihostage Please see https://github.com/playframework/play-grpc/issues/489#issuecomment-1867398720 first, @sentenza was suggesting v1 there.

**Comment by mkurz** [comment]
* https://github.com/playframework/play-grpc/releases/tag/0.10.0
* https://github.com/playframework/play-grpc/releases/tag/0.11.0
* https://github.com/playframework/play-grpc/releases/tag/0.12.0
